### PR TITLE
MGDAPI-6113 Fix removeGrafanaOperatorAlerts() to work for Internal rhoam  (rhmi)

### PR DESCRIPTION
**Pull request by valerymo**
# Issue link
Jira: https://issues.redhat.com/browse/MGDAPI-6113

# What
Fix removeGrafanaOperatorAlerts method to work for Internal rhoam (rhoami)

# Verification steps
Please see related PR: https://github.com/integr8ly/integreatly-operator/pull/3398

### CODE REVIEW

File: pkg/products/grafana/prometheusRules.go [hunk 1]
```diff
@@ -60,14 +61,22 @@ func (r *Reconciler) newAlertReconciler(logger l.Logger, installType string, nam
 }
 
 func (r *Reconciler) removeGrafanaOperatorAlerts(installType string, ctx context.Context, apiClient k8sclient.Client) error {
-	oboNamespace := "redhat-" + resources.InstallationNames[installType] + "-operator-observability"
+	installationName := resources.InstallationNames[installType]
```
Comment (MStokluska) [inline pkg/products/grafana/prometheusRules.go:64]: This looks a bit overcomplicated, WDYT if in removeGrafanaOperatorAlerts we would accept "nsPrefix" string, and where the func is called we can get the nsPrefix from "installation.Spec.NamespacePrefix" then annotate the nsPrefix with "-operator-observability"

Comment (valerymo) [reply pkg/products/grafana/prometheusRules.go]: sure, thanks a lot for this comment.  Updated.

**Comment by valerymo** [comment]
@MStokluska  . Update retested on cluster vmo03. Old alerts - deleted. Thank you for review!

**Comment by trepel** [comment]
I ran the upgrade scenario and it went ok.
/lgtm

**Pull request merged**